utils/validate_images.js

```diff
@@ -25,51 +25,52 @@ const ALLOWED_IMG_EXT = ['.png', '.jpeg', '.jpg', '.svg'];
 const validateImageCounts = (quickstartDirs) => {
   const screenshotDirectories = [];
   const imagesDirectories = [];
-  quickstartDirs
-    .forEach((quickstart) => {
-      const quickstartDirName = path.dirname(quickstart);
-      // get all images for a quickstart
-      const imagePaths = glob.sync(
-        path.resolve(quickstartDirName, '**/*.+(png|jpeg|jpg|svg)')
-      );
-      const quickstartConfig = readQuickstartFile(quickstart).contents[0];
-      const quickstartName = quickstartConfig.name;
-      const logoPath = quickstartConfig.logo
-        ? path.resolve(quickstartDirName, quickstartConfig.logo)
-        : null;
-
-      // Max images is per dashboard so we need to account for this by getting the number of dashboards
-      const dashboardCount = glob.sync(
-        path.resolve(quickstartDirName, 'dashboards/*.json')
-      ).length;
-
-      const screenshotPaths = imagePaths.filter(
-        (p) => p !== logoPath && !p.includes(quickstartName + DASHBOARD_IMAGES_PATH)
-      );
-
-      const dashboardImagePaths = imagePaths.filter(
-        (p) => p !== logoPath && p.includes(quickstartName + DASHBOARD_IMAGES_PATH)
-      );
-
-      // Each dashboard is allowed MAX_NUM_IMG dashboards
-      if (screenshotPaths.length > (MAX_NUM_IMG * dashboardCount)) {
-        screenshotDirectories.push({
-          folder: quickstartDirName,
-          dashboardCount,
-          imageCount: screenshotPaths.length,
-          maxImages: MAX_NUM_IMG * dashboardCount,
-        });
-      }
-
-      if (dashboardImagePaths.length > (MAX_NUM_IMG * dashboardCount)) {
-        imagesDirectories.push({
-          folder: quickstartDirName + DASHBOARD_IMAGES_PATH,
-          dashboardCount,
-          imageCount: dashboardImagePaths.length,
-          maxImages: MAX_NUM_IMG * dashboardCount,
-        });
-      }
-    });
+  quickstartDirs.forEach((quickstart) => {
+    const quickstartDirName = path.dirname(quickstart);
+    // get all images for a quickstart
+    const imagePaths = glob.sync(
+      path.resolve(quickstartDirName, '**/*.+(png|jpeg|jpg|svg)')
+    );
+    const quickstartConfig = readQuickstartFile(quickstart).contents[0];
+    const quickstartName = quickstartConfig.name;
+    const logoPath = quickstartConfig.logo
+      ? path.resolve(quickstartDirName, quickstartConfig.logo)
+      : null;
+
+    // Max images is per dashboard so we need to account for this by getting the number of dashboards
+    const dashboardCount = glob.sync(
+      path.resolve(quickstartDirName, 'dashboards/*/*.json')
+    ).length;
```

Comment:
i imagine this is now returning 0, so every quickstart is failing. 

i think we could:
* modify the syntax to any json file underneath dashboards/. This would work in both scenarios, despite not matching the strictest definition of where a dashboard can live.
* modify the workflow / script to not validate images when no content change is made.
* modify content in the same PR 🙈

Reply:
@nr-kkenney I like the first one, keeps the image validation backwards compatible. I'll get that added to the PR